test/test_pipeline/test_classification.py

```diff
@@ -654,6 +654,20 @@ def test_pipeline_clonability(self):
             param2 = params_set[name]
             self.assertEqual(param1, param2)
 
+    def test_select_rates_for_classification(self):
```

Comment:
Do we still need this test now that we have two components?

Reply:
Not anymore, removed.